src/vsg/threading/Affinity.cpp

```diff
@@ -46,7 +46,7 @@ static void win32_setAffinity(HANDLE tid, const vsg::Affinity& affinity)
 
 void vsg::setAffinity(std::thread& thread, const Affinity& affinity)
 {
-    win32_setAffinity(thread.native_handle(), affinity);
+    win32_setAffinity((HANDLE)thread.native_handle(), affinity);
```

Comment:
Would be better to use a static_cast<HANDLE>(thread).  Could you try this and see if it builds OK?

Reply:
No, now I get 
```
[  123s] /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-0.1.4.20210909.git+7847d18a/src/vsg/threading/Affinity.cpp: In function 'void vsg::setAffinity(std::thread&, const vsg::Affinity&)':
[  123s] /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-0.1.4.20210909.git+7847d18a/src/vsg/threading/Affinity.cpp:49:65: error: invalid static_cast from type 'std::thread::native_handle_type' {aka 'unsigned int'} to type 'HANDLE' {aka 'void*'}
[  123s]    49 |     win32_setAffinity(static_cast<HANDLE>(thread.native_handle()), affinity);
[  123s]       |                                                                 ^
[  123s] make[2]: *** [src/vsg/CMakeFiles/vsg.dir/build.make:1498: src/vsg/CMakeFiles/vsg.dir/threading/Affinity.cpp.obj] Error 1
[  123s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/VulkanSceneGraph-0.1.4.20210909.git+7847d18a'
[  123s] make[2]: *** Waiting for unfinished jobs....
```